ios/Plugin/Plugin.swift

```diff
@@ -1,17 +1,51 @@
 import Foundation
 import Capacitor
+import Sentry
 
-/**
- * Please read the Capacitor iOS Plugin Development Guide
- * here: https://capacitorjs.com/docs/plugins/ios
- */
 @objc(SentryCapacitor)
 public class SentryCapacitor: CAPPlugin {
-
-    @objc func echo(_ call: CAPPluginCall) {
-        let value = call.getString("value") ?? ""
-        call.success([
-            "value": value
-        ])
+    
+    @objc public func startWithOptions(capOptions: CAPPluginCall) {
```

Comment:
@bruno-garcia is this what you had in mind?

Reply:
yeah! awesome